src/gui/uithememanager.cpp

```diff
@@ -42,9 +43,16 @@
 
 namespace
 {
-    const QString ICONS_DIR = QStringLiteral(":icons/");
-    const QString THEME_ICONS_DIR = QStringLiteral(":uitheme/icons/");
-    const QString CONFIG_FILE_NAME = QStringLiteral(":uitheme/config.json");
+    const QString CONFIG_FILE_NAME = QStringLiteral("config.json");
+    const QString DEFAULT_ICONS_DIR = QStringLiteral(":icons/");
+    const QString STYLESHEET_FILE_NAME = QStringLiteral("stylesheet.qss");
+
+    // Directory used by stylesheet to reference internal resources
+    // f.e `icon: url(:/uitheme/file.svg)` will be expected to
```

Comment:
I don't think "f.e" is a valid abbreviation, just write out the full word.

Reply:
I think it is but whatever